<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Schema Name</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="attrNumber">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="agent_name_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Publish Schema</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Schema Version</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="schemaVersion">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>example: 49.92.26</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="schemaName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>employee_schema</string>
   </property>
   <property name="placeholderText">
    <string>example: employee_schema</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Number of Schema Atrritbutes</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="schemaAttributes">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>131</width>
     <height>311</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string extracomment="Your attributes">your_attribute1</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>470</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
